test/e2e/apps/job.go

```diff
@@ -361,6 +366,64 @@ var _ = SIGDescribe("Job", func() {
 			framework.ExpectEqual(pod.Status.Phase, v1.PodFailed)
 		}
 	})
+
+	ginkgo.It("should verify changes to a job status", func() {
+
+		ns := f.Namespace.Name
+		jClient := f.ClientSet.BatchV1().Jobs(ns)
+
+		ginkgo.By("Creating a job")
+		job := e2ejob.NewTestJob("notTerminate", "suspend-false-to-true", v1.RestartPolicyNever, parallelism, completions, nil, backoffLimit)
+		job.Spec.Suspend = pointer.BoolPtr(false)
```

Comment:
no need for this line (it's false by default)

Reply:
Removed.